test/shared_utilities/space_varying_inputs.jl

```diff
@@ -0,0 +1,75 @@
+using Test
+import ClimaLand
+using ClimaLand.SpaceVaryingInputs: SpaceVaryingInput
+using ClimaLand: Domains
+
+using ClimaCore: Fields
+using ClimaComms
+
+device = ClimaComms.device()
+context = ClimaComms.SingletonCommsContext(device)
+
+@testset "SpaceVaryingInput" begin
```

Comment:
Could you add a comment for which cases this tests (0D and 2D without structs, I think)?

Reply:
added!